Samples~/SampleGame/Assets/Scripts/Server/GameLiftServer.cs

```diff
@@ -203,12 +214,13 @@ private ProcessParameters CreateProcessParameters()
                 _logger.Write(":) GAMELIFT HEALTH CHECK REQUESTED (HEALTHY)");
                 return true;
             },
-            _port, // tell the GameLift service which port to connect to this process on.
-                   // unless we manage this there can only be one process per server.
+            // tell the GameLift service which port to connect to this process on.
+            _port,
+            // unless we manage this there can only be one process per server.
```

Comment:
I think you misunderstood, this line is part of the `_port` comment

Reply:
Fixed :)